data/translations/ru/state-data.json

```diff
@@ -0,0 +1,261 @@
+  },
+  "ky": {
+    "default_ip_deadline": "За 29 дней до дня выборов",
+    "default_online_deadline": "За 29 дней до дня выборов",
+    "default_mail_postmarked_deadline": "Должен быть проштемпелёван за 29 дней до дня выборов"
+  },
+  "la": {
+    "default_ip_deadline": "За 30 дней до дня выборов",
+    "default_online_deadline": "За 20 дней до дня выборов",
+    "default_mail_postmarked_deadline": "Должен быть проштемпелёван за 30 дней до дня выборов"
+  },
+  "me": {
+    "default_ip_deadline": "Доступно до дня выборов включительно",
+    "default_mail_received_deadline": "Должен быть получен за 21 день до дня выборов"
+  },
+  "md": {
+    "default_ip_deadline": "Доступно до дня выборов включительно",
+    "default_online_deadline": "За 21 дней до дня выборов",
+    "default_mail_postmarked_deadline": "Должен быть проштемпелёван за 21 дней до дня выборов"
+  },
+  "ma": {
+    "default_ip_deadline": "За 10 дней до дня выборов",
+    "default_online_deadline": "За 10 дней до дня выборов",
+    "default_mail_postmarked_deadline": "Должен быть проштемпелёван за 10 дней до дня выборов"
+  },
+  "mi": {
+    "default_ip_deadline": "Доступно до дня выборов включительно",
+    "default_online_deadline": "За 15 дней до дня выборов",
+    "default_mail_postmarked_deadline": "Должен быть проштемпелёван за 15 дней до дня выборов"
+  },
+  "mn": {
+    "default_ip_deadline": "Доступно до дня выборов включительно",
+    "default_online_deadline": "За 21 день до дня выборов",
+    "default_mail_received_deadline": "Должен быть получен за 21 день до дня выборов"
+  },
+  "ms": {
+    "default_ip_deadline": "За 30 дней до дня выборов",
+    "default_mail_postmarked_deadline": "Должен быть проштемпелёван за 30 дней до дня выборов"
+  },
+  "mo": {
+    "default_ip_deadline": "За 27 дней до дня выборов",
+    "default_online_deadline": "За 27 дней до дня выборов",
+    "default_mail_postmarked_deadline": "Должен быть проштемпелёван за 27 дней до дня выборов"
+  },
+  "mt": {
+    "default_ip_deadline": "Доступно до дня выборов включительно",
+    "default_mail_postmarked_deadline": "Должен быть проштемпелёван за 30 дней до дня выборов"
+  },
+  "ne": {
+    "default_ip_deadline": "За 11 дней до дня выборов",
+    "default_online_deadline": "За 18 дней до дня выборов",
+    "default_mail_postmarked_deadline": "Должен быть проштемпелёван за 18 дней до дня выборов"
+  },
+  "nv": {
+    "default_ip_deadline": "Доступно до дня выборов включительно",
+    "default_online_deadline": "За 5 дней до дня выборов",
+    "default_mail_postmarked_deadline": "Должен быть проштемпелёван за 28 дней до дня выборов"
+  },
+  "nh": {
+    "default_ip_deadline": "Доступно до дня выборов включительно",
+    "default_mail_received_deadline": "Должен быть получен за 13 дней до дня выборов"
+  },
+  "nj": {
+    "default_online_deadline": "За 21 день до дня выборов",
+    "default_mail_received_deadline": "Должен быть получен за 13 дней до дня выборов",
```

Comment:
Spot checking this is the wrong number of days. Can you recheck this whole document against the english version of the site to make sure there is no regressions to the number of days.